Makefile

```diff
@@ -828,6 +828,7 @@ internal-update-repo-templates-%: UPDATE_REPO_SUBDIR = $(TARGET_REPO)
 # and the actual code
 # this is executed for every (DIST,PACKAGE_SET,COMPONENT) combination
 internal-update-repo-%:
+ifeq ($(MAKEREPO),YES)
```

Comment:
This (and the other) variable needs default value, otherwise calling make update-repo-* manually won't work (will trigger `$(error ...)` below).

But also, we use `0` / `1` in other places for boolean settings, it should be this way here too, for consistency.